DABI.js

```diff
@@ -0,0 +1,482 @@
+				"title": "\"Was ihr wollt!\" Nutzungsgesteuerter Einkauf von Medien an der Staatsbibliothek zu Berlin",
+				"creators": [
+					{
+						"firstName": "Janin",
+						"lastName": "Taubert",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014",
+				"issue": "3",
+				"libraryCatalog": "DABI",
+				"pages": "79-81",
+				"publicationTitle": "Bibliotheks-Magazin",
+				"shortTitle": "Was ihr wollt",
+				"url": "http://staatsbibliothek-berlin.de/fileadmin/user_upload/zentrale_Seiten/ueber_uns/pdf/Bibliotheksmagazin/Bibliotheksmagazin_3-2014.pdf",
+				"volume": "9",
+				"attachments": [
+					{
+						"title": "DABI Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [
+					{
+						"tag": "Benutzerorientierter Bestandsaufbau"
+					},
+					{
+						"tag": "Benutzerorientierung"
+					},
+					{
+						"tag": "Berlin"
+					},
+					{
+						"tag": "Bestand"
+					},
+					{
+						"tag": "Bestandsaufbau"
+					},
+					{
+						"tag": "Bibliothekswesen"
+					},
+					{
+						"tag": "Demand Driven Acquisition (DDA)"
+					},
+					{
+						"tag": "E-Book"
+					},
+					{
+						"tag": "Evidence Based Selection (EBS)"
+					},
+					{
+						"tag": "Kundenorientierter Bestandsaufbau"
+					},
+					{
+						"tag": "Patron Driven Acquisition (PDA)"
+					},
+					{
+						"tag": "Purchase On Demand (POD)"
+					},
+					{
+						"tag": "Staatsbibliothek zu Berlin - Preußischer Kulturbesitz (SBB PK)"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://dabi.ib.hu-berlin.de/cgi-bin/dabi/vollanzeige.pl?artikel_id=5676&modus=html",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Anpassung der Personalstruktur der Fachhochschulbibliotheken in Nordrhein-Westfalen an die Erfordernisse der neunziger Jahre",
+				"creators": [],
+				"date": "1992",
+				"issue": "1",
+				"libraryCatalog": "DABI",
+				"pages": "364-372",
+				"publicationTitle": "Mitteilungsblatt des Verbandes der Bibliotheken des Landes Nordrhein-Westfalen",
+				"shortTitle": "Anpassung der Personalstruktur der Fachhochschulbibliotheken in",
+				"volume": "4",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Nordrhein-Westfalen"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://dabi.ib.hu-berlin.de/cgi-bin/dabi/vollanzeige.pl?artikel_id=9481&modus=html",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "\"Das bibliophile Flaggschiff Bayerns\": Auszug aus der Rede des damaligen Ministerpräsidenten Dr. Günther Beckstein",
+				"creators": [
+					{
+						"firstName": "Günther",
+						"lastName": "Beckstein",
+						"creatorType": "author"
+					}
+				],
+				"date": "2009",
+				"issue": "1",
+				"libraryCatalog": "DABI",
+				"pages": "46",
+				"publicationTitle": "Bibliotheksforum Bayern",
+				"shortTitle": "Bibliophile Flaggschiff Bayerns",
```

Comment:
I also find this quite awkward. It might serve individual preferences (?), but for the most common usage of short titles, i.e. as short citation markers, I think this works much better with article.

Reply:
Dito. And: You are right.